.changeset/popular-pens-join.md

```diff
@@ -0,0 +1,6 @@
+---
+"@sap-ux-private/preview-middleware-client": patch
+'@sap-ux/preview-middleware': patch
```

Comment:
Do we need a changeset for this branch? Code is not affected. Only if we need to trigger release build immediately

Reply:
No we don't need a new version. Currently its just documentation for development purposes.